react/components/ProfileContents/index.js

```diff
@@ -29,6 +29,7 @@ export default class ProfileContents extends Component {
     per: 12,
     hasMore: true,
     q: null,
+    seed: Date.now(),
```

Comment:
Won't the seed be different between ssr and the client-side paging?

Reply:
Ah yeah, good point. Hm, any ideas?

Reply:
Set it as a cookie or push it into the query params so that it's at a routing level.